react/react1/week1/todo-list/src/componate/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import TodoList from './TodoList';
+import Header from './Header';
+import Footer from './Footer';
+import todoList from '../Todo';
+
+function createTodo(todoList) {
+    return (
+      <TodoList
```

Comment:
if you're mapping through the array you don't get a list, you get a list item, so the component naming should follow the logic behind it.